src/main/java/org/lookout_studios/meals_management_system/meals_management_system/mysqlConnection.java

```diff
@@ -0,0 +1,30 @@
+package org.lookout_studios.meals_management_system.meals_management_system;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class MysqlConnection {
+    public void sqlConnection() {
+        String url = "jdbc:mysql://localhost:3306/test"; /* Change `test` to desired database.*/
+        String username = "root";
+        String password = "";
```

Comment:
Do not include credentials in the source code! Create a separate config file for that and DO NOT include its contents in the repository